src/store/modules/application/getters.js

```diff
@@ -46,6 +48,11 @@ export default {
     return { type, definition };
   },
 
+  currentComponentDefinition(state, getters) {
+    const { definition } = getters['currentComponent'];
```

Comment:
Why do we create a constant for it before returning it?

Reply:
good point. fixing it.